tests/core/test_tracked.py

```diff
@@ -0,0 +1,79 @@
+import concurrent.futures
+
+import lamindb as ln
+import pandas as pd
+import pytest
+
+
+@ln.tracked()
+def process_chunk(chunk_id: int) -> str:
+    # Create a simple DataFrame
+    df = pd.DataFrame(
+        {"id": range(chunk_id * 10, (chunk_id + 1) * 10), "value": range(10)}
+    )
+
+    # Save it as an artifact
+    key = f"chunk_{chunk_id}.parquet"
+    artifact = ln.Artifact.from_df(df, key=key).save()
+    return artifact.key
+
+
+def test_tracked_parallel():
```

Comment:
There's a lot of prints here that clutter when we run `pytest -s`. Can we remove them again?

Reply:
Don't you want to see the prints when running `pytest -s`? 😅

Reply:
Yes but only the prints that I care about and not prints in many tests haha. Sure - fair game if I only run this test, but if we start adding prints to many tests, it's not the nicest UX.